cypress/plugins/index.js

```diff
@@ -20,3 +20,18 @@ module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
 }
+
+/// <reference types="cypress" />
+
+const wp = require('@cypress/webpack-preprocessor')
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+module.exports = (on, config) => {
+  const options = {
+    webpackOptions: require('../../webpack.config'),
+  }
+
+  on('file:preprocessor', wp(options))
+}
```

Comment:
This section of code should replace the lines 15 to 22. So you can remove those lines (15 to 22).

Reply:
Understood. Already modified